optimum/exporters/onnx/model_configs.py

```diff
@@ -1085,6 +1085,10 @@ def outputs(self) -> Dict[str, Dict[int, str]]:
         return outputs
 
 
+class OwlV2OnnxConfig(OwlViTOnnxConfig):
+    DEFAULT_ONNX_OPSET = 14
```

Comment:
Can you add a comment as to why this opset is needed?

Reply:
Removed since it's not actually needed.